src/ts/models.ts

```diff
@@ -30,30 +30,32 @@ export class Config implements IConfig {
 }
 
 export class RoundResultDTO {
+  questionUUId: string;
```

Comment:
Did… did you write it `UUId`? That is wrong.
It should be `UUID`, while there is `ID` present, there is more than one word here:
`Universally Unique IDentifier`…